src/Workspaces/Core/Portable/Diagnostics/DiagnosticAnalysisResult.cs

```diff
@@ -148,15 +120,6 @@ public static DiagnosticAnalysisResult CreateFromBuilder(DiagnosticAnalysisResul
             builder.DocumentIds);
     }
 
-    // aggregated form means it has aggregated information but no actual data.
-    public bool IsAggregatedForm => _syntaxLocals == null;
```

Comment:
this was just staggering confusing as to what this meant.  it was basically a way to store higher up that parts of hte data were not computed while other parts were.  but this coudl NEVER happen, as this was only ever created from ForceAnalyzeProjectAsync which always computed all data...